src/components/dashboard/GapChart.vue

```diff
@@ -140,7 +146,10 @@ const spec = computed(() => {
         encode: {
           enter: {
             x: { scale: "xscale", field: "pct" },
-            x2: { scale: "xscale", value: expected.value },
+            x2: {
+              scale: "xscale",
+              value: `datum.population > 0 ? ${expected.value} : 0`,
```

Comment:
I think you'll want to add `strokeWidth` here and on the bar and make it conditional on the population as well - that should get rid of the faint amount of mark left

Reply:
I changes this line back. And instead used a test on the stoke